astroquery/astrometry_net/core.py

```diff
@@ -412,6 +415,11 @@ def solve_from_image(self, image_file_path, *, force_image_upload=False,
                                          cache=False,
                                          files={'file': f})
         else:
+            warning_msg = "Removing photutils functionality to obtain extracted positions list from " \
+                          "AstoromertyNetClass.solve_from_source_list. Users will need to " \
```

Comment:
```suggestion
                          "AstrometryNetClass.solve_from_source_list. Users will need to " \
```

Reply:
Hey @keflavich thank you for the feedback. I fixed the typo in [4e23354](https://github.com/astropy/astroquery/pull/3067/commits/4e2335417ecc2d04c319e071422742b47846c28b)